presto-main/src/main/java/io/prestosql/sql/planner/LogicalPlanner.java

```diff
@@ -162,6 +163,11 @@ public Plan plan(Analysis analysis)
     }
 
     public Plan plan(Analysis analysis, Stage stage)
+    {
+        return plan(analysis, stage, analysis.getStatement() instanceof Explain || isCollectPlanStatistics(session));
+    }
+
+    public Plan plan(Analysis analysis, Stage stage, boolean collectStatsAndCosts)
```

Comment:
here, "collect plan statistics" gets renamed to "collect stats and costs"

can we use one name everywhere?